src/main/java/net/minecraftforge/common/crafting/IRecipeContainer.java

```diff
@@ -0,0 +1,15 @@
+package net.minecraftforge.common.crafting;
```

Comment:
Please include the forge license header comment on new files.

Reply:
whoops, sorry

Reply:
done